CSharpExtractor/CSharpExtractor/Extractor/Program.cs

```diff
@@ -44,12 +44,14 @@ static void Main(string[] args)
 
             results = files.AsParallel().WithDegreeOfParallelism(options.Threads).SelectMany(filename => ExtractSingleFile(filename, options));
 
-            //create one file per job, write first time, append all following times
             StreamWriter sw = new StreamWriter(options.OFileName, append: true);
             foreach (var res in results)
             {
                 sw.WriteLine(res);
             }
+            sw.Flush();
```

Comment:
how about the IDisposable way of:
`using (StreamWriter sw = new StreamWriter(...) {
   foreach (.. ) 
   {
      sw.WriteLine(...);
   }
}
`

Reply:
and then you won't need those 3 lines of flushing, closing, and disposing.

Reply:
Done